### PR TITLE
Add Ren'Py language and grammars

### PR DESCRIPTION
**Pull request by williamd1k0**
Hi! I would like to add the *Ren'Py* language, a programming language derived from Python utilized by a game engine with the same name. *Official site: http://www.renpy.org/*
At first i just wanted to ask to add the `.rpy` extension to Python's scope, but as there are some notable differences i decided to add the grammars file too, that i made myself.
I have some questions before merging:
* I left the language in the Python group (`group: Python`), but it has a different scope so i am not sure if i it should left it that way.
* I would like to know if the changes are sufficient for the language to show on Github's repository detector.

Link of the search using the extension `.rpy`: https://github.com/search?utf8=%E2%9C%93&q=extension%3Arpy+NOT+nothack

I hope everything is under the parameters of "CONTRIBUTING.md", thanks! :laughing:

### CODE REVIEW

File: samples/RenPy/example.rpy [hunk 1]
```diff
@@ -0,0 +1,1427 @@
+###     Demo Script Example     ###
```
Comment (pchaigno) [inline samples/RenPy/example.rpy]: The sample folder needs to be named exactly like the language ;)

**Comment by pchaigno** [comment]
Being in the Python group means that Ren'Py files will be counted as Python files in the statistics. It's mostly useful to define a different grammar. So, I think it's appropriate in Ren'Py's case.

**Comment by arfon** [comment]
:+1: thanks for a nice Pull Request @williamd1k0 - this will be live in the next release of Linguist (sometime this week).

**Comment by williamd1k0** [comment]
Oh, thank you! It's a pleasure! :+1:

**Pull request merged**